explorer/interpreter/interpreter.cpp

```diff
@@ -1283,15 +1370,14 @@ auto Interpreter::StepExp() -> ErrorOr<Success> {
       if (act.pos() == 0) {
         return todo_.Spawn(
             std::make_unique<ExpressionAction>(&access.object()));
-      } else {
-        ElementPath::Component base_elt(&access.element(), std::nullopt,
-                                        std::nullopt);
-        const Value* value = act.results()[0];
-        CARBON_ASSIGN_OR_RETURN(Nonnull<const Value*> base_value,
-                                value->GetElement(arena_, ElementPath(base_elt),
-                                                  exp.source_loc(), value));
-        return todo_.FinishAction(base_value);
       }
```

Comment:
Probably unwanted `} else {` removal

Reply:
Sorry I did this by mistake, reverting it in next commit.